### PR TITLE
Use signal_t from libswiftnav and libsbp.

### PR DESCRIPTION
**Pull request by gsmcmullin**

### CODE REVIEW

**Comment by gsmcmullin** [comment]
retest this please

**Comment by gsmcmullin** [comment]
retest this please

**Comment by jacobmcnamee** [comment]
See inline comments. Mostly just var names.

File: src/manage.c [hunk 1]
```diff
@@ -326,6 +336,7 @@ static void manage_acq()
   u8 prn = choose_prn();
   if (prn == (u8)-1)
     return;
+  gnss_signal_t sid = {.sat = prn};
```
Comment (jacobmcnamee) [inline src/manage.c]: set constellation = CONSTELLATION_GPS and band = BAND_L1

Comment (gsmcmullin) [inline src/manage.c]: I couldn't be bothered to fix this now.  In the next SBAS PR, `choose_prn` will become `choose_sid` and will return the sid with constellation and band filled in.

**Comment by jacobmcnamee** [comment]
My only other concern is receiving SBP messages with sat >= 32 and causing an array overrun. Probably not a huge deal, but regardless it will be resolved when full signal_t support is implemented.

**Pull request merged**